packages/test-runner-webdriver/src/IFrameManager.ts

```diff
@@ -155,6 +155,13 @@ export class IFrameManager {
     return this.driver.keys(keys);
   }
 
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  async performActions(_: string, _actions: object[]) {
+    console.warn(
+      'Unsupported operation. In order to use device actions in Webdriver, set "concurrency" to 1.',
+    );
```

Comment:
I think `console.warn` isn't quite right here. Should we throw an error instead?

Reply:
Done.